src/pricing/ar_data_price_network_estimator.ts

```diff
@@ -0,0 +1,86 @@
+import { GatewayOracle } from './gateway_oracle';
+import type { ArweaveOracle } from './arweave_oracle';
+import { AbstractARDataPriceAndCapacityEstimator } from './ar_data_price_estimator';
+import { AR, ByteCount, Winston, ArDriveCommunityTip } from '../types';
+
+const byteCountOfChunk = new ByteCount(Math.pow(2, 10) * 256); // 256 KiB
```

Comment:
```suggestion
const byteCountPerChunk = new ByteCount(Math.pow(2, 10) * 256); // 256 KiB
```
You'd need to change it in all usage locations.

Reply:
0dd92a8

Reply:
Looks like that change got made in the wrong file.